posthog/models/event/event.py

```diff
@@ -48,6 +48,12 @@ def __init__(self, tag: str, direct_descendant: bool, escape_slashes: bool):
             # Strip all slashes that are not followed by another slash
             self.data["attr_class__contains"] = [self._unescape_class(p) if escape_slashes else p for p in parts[1:]]
             tag = parts[0]
+        if "#" in tag:
```

Comment:
Just convert the #selector its [id="blah"] equivalent 🤯 

🙌 that's really elegant 💖